vnext/ReactUWP/Modules/Animated/AnimationDriver.cpp

```diff
@@ -64,8 +64,15 @@ void AnimationDriver::StartAnimation() {
 void AnimationDriver::StopAnimation() {
   if (const auto animatedValue = GetAnimatedValue()) {
     animatedValue->PropertySet().StopAnimation(ValueAnimatedNode::s_offsetName);
-    m_endCallback(
+    animatedValue->RemoveActiveAnimation(m_id);
+
+    if (m_scopedBatch)
+    {
+      m_endCallback(
```

Comment:
>m_endCallback [](start = 6, length = 13)

Should maybe ensure that we have a non-null callback before we attempt to call it..